ui/package.json

```diff
@@ -32,10 +33,14 @@
     "eslint-plugin-prettier": "^5.1.3",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-react-refresh": "^0.4.3",
+    "execa": "^8.0.1",
+    "happy-dom": "^13.3.1",
     "prettier": "^3.2.4",
+    "puppeteer": "^21.9.0",
```

Comment:
hm. Let's not forget that dapp-offer-up is supposed to be *simple*. I'm a little afraid that readers are going to get overwhelmed by some of this stuff. On the other hand, it's the only sane way to be sure we're not breaking things.

Reply:
Good points. In [d3b726c](https://github.com/Agoric/dapp-offer-up/pull/57/commits/d3b726c0b540adf439102406a65d675969a6f1bf) I moved the test to `/test` so it's less prominent. Also have it only running on `yarn test:e2e`, instead of `yarn test`.


Maybe this sort of test is best for a [`react-scripts`-esque](https://www.npmjs.com/package/react-scripts) library that this eventually depends on.